File: Selenium/Project/Activity5.java
package activities;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Test;

import io.github.bonigarcia.wdm.WebDriverManager;

public class Activity5 {
			//Declare WebDriver object
	    	static WebDriver driver;
	    	
	    	@BeforeClass
	    	public void beforeMethod(){
	    		// Set up Firefox driver
	            WebDriverManager.firefoxdriver().setup();
	            // Create a new instance of the Firefox driver
	            driver = new FirefoxDriver();
	    		// Open the browser
	    		driver.get("https://alchemy.hguy.co/lms/");
	    	}
	    	
	    	@Test
	    	public void findNavigationBar() {
	    	
	    		//find the my account navigation bar
	    		driver.findElement(By.xpath("//a[contains(text(),'My Account')]")).click();
	    		// Print the title of the new page
	    		System.out.println("My account page title: " + driver.getTitle());
	    	}
			
			  @AfterClass public void afterMethod() {
				  //close the browser
				  driver.quit();
			  } 
}

File: Selenium/Project/Activity5.py
# Import webdriver from selenium
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager
from selenium.webdriver.support.wait import WebDriverWait

# Set up the Firefox Driver with WebDriverManger
service = FirefoxService(GeckoDriverManager().install())

# Start the Driver
with webdriver.Firefox(service=service) as driver:
     # Navigate to the URL
    driver.get("https://alchemy.hguy.co/lms/")

    My_Account = driver.find_element(By.XPATH, "//a[contains(text(),'My Account')]")
    My_Account.click()
    Page_tittle = driver.title
    print("Title of the page : ", Page_tittle)
    assert Page_tittle == "My Account – Alchemy LMS"